tests/test_trainer.py

```diff
@@ -49,20 +49,22 @@ def _model_base_test( # pylint: disable=too-many-arguments, too-many-locals
         hdf5_path=train_hdf5_path,
         node_features=node_features,
         edge_features=edge_features,
-        target=target,
-        task = task,
         clustering_method=clustering_method,
-        target_transform = target_transform)
+        target=target,
+        target_transform = target_transform,
+        task = task
+        )
 
     if val_hdf5_path is not None:
         dataset_val = GraphDataset(
             hdf5_path=val_hdf5_path,
             node_features=node_features,
             edge_features=edge_features,
-            target=target,
-            task = task,
             clustering_method=clustering_method,
-            target_transform = target_transform)
+            target=target,
```

Comment:
Same as above, in validation and testing case we need to match the code with the train and dataset_train logic